Update README.md for guidance, starting somewhere

barriebyron opened the pull request:
start a draft README to provide guidance, per https://github.com/cosmos/advocates/issues/15

Commit:
Update README.md

start a draft README to provide guidance, per https://github.com/cosmos/advocates/issues/15

hxrts commented on README.md:
```diff
@@ -1,7 +1,29 @@
+As we scale our content teams, issue triage is a fluid process that we're still working out. Thank you for your patience. Thank you for caring.
+
+- For Cosmos SDK and Starport docs, tag @barriebyron 
+- For Cosmos SDK tutorials, tag @toschdev 
```
some things not covered that should have owners:
- IBC protocol website https://ibcprotocol.org/
- rust IBC & hermes relayer (should be andy)
- typescript relayer
- tendermint core
- tendermint spec repo
- IBC spec repo
- cosmjs

hxrts commented:
ty @barriebyron !

barriebyron commented:
I don't know who and what to add here, but Elias from B9 can take it from here.

Commit:
Update README.md

clarify project repo first for technical content issues, let's use the advocates repo for issues that are wider in scope and unrelated to specific docs

hxrts commented:
btw, I don't have perms to merge, but this LGTM to start

barriebyron commented:
@okwme I would tag Elias for his review and attention if I knew his GitHub username. Can you tag him here?

soniasingla commented:
@barriebyron is it ready to merge? :)

barriebyron commented:
@soniasingla yes, ready, but we require 2 approvals in this repo. @8bitpal can approve?

barriebyron commented:
maybe @hxrts can approve so we can merge?
